components/Account.php

```diff
@@ -338,7 +338,22 @@ public function onRegister()
              * Automatically activated or not required, log the user in
              */
             if ($automaticActivation || !$requireActivation) {
-                Auth::login($user);
+                /*
+                * Login remember mode
+                */
+                switch ($this->rememberLoginMode()) {
+                    case UserSettings::REMEMBER_ALWAYS:
+                        $remember = true;
+                        break;
+                    case UserSettings::REMEMBER_NEVER:
+                        $remember = false;
+                        break;
+                    case UserSettings::REMEMBER_ASK:
+                        $remember = (bool) array_get($data, 'remember', false);
+                        break;
+                }
+
+                Auth::login($user, $remember);
```

Comment:
@bernardotavares Could you change this code block to the following, and test it out for us.
```suggestion
                Auth::login($user, $this->rememberLoginMode() === UserSettings::REMEMBER_ALWAYS);
```

Reply:
That way it's only true if the user specifically requested and we don't even add a single line of code. Love it. Will test it out.